pkg/model/stack_serializer_test.go

```diff
@@ -867,6 +867,9 @@ func Test_validateVolumesUnmarshalling(t *testing.T) {
 	}
 	defer os.RemoveAll("test_volume_relative_path_found")
 	file, err := os.Create("test-file")
+	if err != nil {
```

Comment:
What's the linting error here? This seems to modify our test assertions.

Reply:
We were not handling the error that came back from the `ok.Create` function (I think it was `unused`)

Reply:
If we don't want to change the test check, other possible solution could be replace `err` by `_` and don't check the error.

Anyways, if this is some kind of precondition in our test, I think it is ok to check the error and fail

Reply:
@ifbyol Exactly, as it's a precondition I would fail the test if we cannot create the file